pkg/rulefmt/testdata/test.yaml

```diff
@@ -0,0 +1,59 @@
+version: 1
+groups:
+- name: my-group-name
+  interval: 30s   # defaults to global interval
+  rules:
+  - alert: HighErrors
+    expr: |
+      sum without(instance) (rate(errors_total[5m]))
+      / sum without(instance) (rate(requests_total[5m]))
```

Comment:
That kind of constraint goes straight to my worries about nesting one kind of language in another one, especially if the outer one is one that cares about white space like this. I don't know what conclusion to draw from this though, as there might not be a better way.